backend/endpoints/detail_file.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import APIRouter, UploadFile,  status, HTTPException
+from fastapi.responses import FileResponse
+from utils.message_utils import get_message
+from starlette.responses import JSONResponse, Response
+import datetime, zipfile,json
+from endpoints import image
+import requests, zipfile, os
+
+router = APIRouter()
+"""
+	An endpoint that returns the details of a file uplaoded to the given room
+	params:
+		org_id: organization id number
+		room_id: room id number
+        message_id: message id number
+		return:
+			The details of a file uploaded
+
+    org_id :6373eb474746182adae97314
+    6373eb4f4746182adae97316
+    room_id: 6373eb4f4746182adae97316
+    message_id:637fe320601ce3fc5dc739ea
+"""
```

Comment:
That's not where the docstring should be written.
Kindly check the other functions in the codebase.

Reply:
Done